doc/setting_up_sidewalk_environment/requirements.rst

```diff
@@ -33,12 +33,6 @@ Ensure you have the following:
 * A supported Nordic Semiconductor's development kit and, if required, other additional hardware.
   For the list of the required devices per sample, see :ref:`samples_list`.
 
-* Bluetooth wireless interface on your machine or a USB Bluetooth adapter.
-
-   .. note::
-      Bluetooth wireless adapter is used during the device registration to connect the machine with the development kit.
-      For example, you can use a `USB Bluetooth adapter`_ recommended by Amazon.
-
```

Comment:
Should we also remove the `USB Bluetooth adapter` link from the bottom of the article?

Reply:
Yes, link removed